### PR TITLE
bump rsp to 1.4.1103-1

### PR DESCRIPTION
**Pull request by melissa-barca**

### CODE REVIEW

**Comment by melissa-barca** [comment]
Hi @colearendt could you review this? I don't have permission to add reviewers or merge. Thanks!

**Comment by colearendt** [comment]
@melissa-barca thanks!! Has RStudio Server Pro changed where the .deb files are located? 

Do you have access to this build on docker hub? If not, it might be worth asking IT for access to docker hub!

https://hub.docker.com/repository/registry-1.docker.io/rstudio/rstudio-server-pro/builds/fbeafc60-d746-4a34-bba7-3189a217b40b

```
Reading package lists...
[91m % Total % Received % Xferd Average Speed Time Time Time Current
Dload Upload Total Spent Left Speed
0 0 0 0 0 0 0 0 --:--:-- --:--:-- --:--:-- 0[0m[91m 100 330 0 330 0 0 3267 0 --:--:-- --:--:-- --:--:-- 3267
[0m Reading package lists... 0% Reading package lists... 0% Reading package lists... 0% Reading package lists... 6% Reading package lists... 6% Reading package lists... 6% Reading package lists... 6% Reading package lists... 35% Reading package lists... 59% Reading package lists... 59% Reading package lists... 60% Reading package lists... 60% Reading package lists... 70% Reading package lists... 70% Reading package lists... 72% Reading package lists... 72% Reading package lists... 72% Reading package lists... 82% Reading package lists... 82% Reading package lists... 82% Reading package lists... 82% Reading package lists... 82% Reading package lists... 82% Reading package lists... 82% Reading package lists... 82% Reading package lists... 91% Reading package lists... 91% Reading package lists... 93% Reading package lists... 93% Reading package lists... 99% Reading package lists... 99% Reading package lists... 99% Reading package lists... 99% Reading package lists... Done
Building dependency tree... 0% Building dependency tree... 0% Building dependency tree... 50% Building dependency tree... 50% Building dependency tree
Reading state information... 0% Reading state information... 0% Reading state information... Done
[91mFailed to open the software package
The package might be corrupted or you are not allowed to open the file. Check the permissions of the file.
[0mThe command '/bin/sh -c apt-get update --fix-missing && curl -O ${RSP_DOWNLOAD_URL}/rstudio-server-pro-${RSP_VERSION}-amd64.deb && gdebi --non-interactive rstudio-server-pro-${RSP_VERSION}-amd64.deb && rm rstudio-server-pro-${RSP_VERSION}-amd64.deb && apt-get clean && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 1
build hook failed! (1)
```

And this is the base URL we are using, I believe: https://github.com/rstudio/rstudio-docker-products/blob/c5767e2f86dcfa9cd9b699082a33895d10b91378/server-pro/.env#L2

**Comment by melissa-barca** [comment]
Thanks @colearendt I fixed the URL and the tests are now passing.

File: server-pro/Dockerfile [hunk 1]
```diff
@@ -78,8 +78,8 @@ RUN chmod +x /usr/local/bin/startup.sh
 COPY conf/* /etc/rstudio/
 
 # Install RStudio Server Pro --------------------------------------------------#
-ARG RSP_VERSION=1.3.1093-1
-ARG RSP_DOWNLOAD_URL=https://download2.rstudio.org/server/bionic/amd64
+ARG RSP_VERSION=1.4.1103-3
+ARG RSP_DOWNLOAD_URL=https://download2.rstudio.org/server/xenial/amd64
```
Comment (colearendt) [inline server-pro/Dockerfile:82]:
```suggestion
ARG RSP_DOWNLOAD_URL=https://download2.rstudio.org/server/bionic/amd64
```
I _think_ this should still be bionic (it doesn't get used in the build process)

**Pull request merged**